seed/challenges/intermediate-bonfires.json

```diff
@@ -48,7 +48,7 @@
       "title": "Diff Two Arrays",
       "difficulty": "2.01",
       "description": [
-        "Compare two arrays and return a new array with any items not found in both of the original arrays.",
+        "Compare two arrays and return a new array with the elements that appear in only one of the two arrays. In other words, the elements in the new array should contain the union of <code>arr1</code> and <code>arr2</code> minus the intersection of <code>arr1</code> and <code>arr2</code>",
```

Comment:
In light of discussion https://github.com/FreeCodeCamp/FreeCodeCamp/issues/3758
it seems the explanation should mention
**`UniqueValues((arr1 ∪ arr2) - (arr1 ∩ arr2))`** instead of ~~`(arr1 ∪ arr2) - (arr1 ∩ arr2)`~~